pkg/server/routes.go

```diff
@@ -180,8 +180,6 @@ func (s *Server) routes() {
 	// set up GraphQL routes
 	gql := s.router.PathPrefix("/api/graph").Subrouter()
 
-	gql.Use(requirePrincipalMiddleware)
```

Comment:
@StevenWadeOddball I removed the middleware here to allow subscriptions to go through and authenticate during the init function. Regular http requests should still authenticate normally through the `oktaMiddlewareFactory.NewAuthenticationMiddleware` applied to the entire router (line 82).

Reply:
@aterstriep , I think we might need to add some additional safe guards here. Not all queries are protected with @hasRole() directives. 

The effect of which is that data can be queried without being logged into the application. You can test by turning off local authentication in envrc.local --> export LOCAL_AUTH_ENABLED=false

You can update a bruno query to not have auth, or simply use a curl like this
```bash
curl -X POST http://localhost:8080/api/graph/query  -H "Content-Type: application/json"  -d '{"query": "{ systemIntakes(openRequests: true) { id } }"}'  
```

I found that I returned data without auth.